test/functional/messenger-test.js

```diff
@@ -137,12 +146,13 @@ describes.realWin('Messenger', {}, env => {
       source = {
         postMessage: sandbox.spy(),
       };
-      handler({
+      handler(msg({
         origin: 'https://example-sp.com',
+        source,
```

Comment:
Where is this source variable set?

Shouldn't you have two sources to actually have a test for an iframe?

Reply:
The main `source` var is set up in the `beforeEach` of the test. The failure is also tested below - it's the one with the `anotherSource` var.